crates/compiler/builtins/roc/Dict.roc

```diff
@@ -208,3 +210,413 @@ insertIfVacant = \dict, key, value ->
         dict
     else
         Dict.insert dict key value
+
+# We have decided not to expose the standard roc hashing algorithm.
+# This is to avoid external dependence and the need for versioning.
+# For now, it will just live within the Dict file.
+# If we add non-exposed roc standard library files, this should be moved to it's own file.
+# The current implementation is a form of [Wyhash final3](https://github.com/wangyi-fudan/wyhash/blob/a5995b98ebfa7bd38bfadc0919326d2e7aabb805/wyhash.h).
+# It is 64bit and little endian specific currently.
+LowLevelHasher := { originalSeed : U64, state : U64 } has [
+         Hasher {
+             addBytes,
+             addU8,
+             addU16,
+             addU32,
+             addU64,
+             addU128,
+             addI8,
+             addI16,
+             addI32,
+             addI64,
+             addI128,
+             complete,
+             reset,
+         },
+     ]
+
+createLowLevelHasher : { seed ?U64 } -> LowLevelHasher
```

Comment:
Same here - I think `roc format` should format the type annotation as `{ seed ? U64 }`; if not, we should raise bug

Reply:
Yeah, this one looks like a `roc format` bug. Will reformat the file and double check.

Reply:
Yeah, a `roc format` bug. I think we should just submit this PR with the bug and then point to it in an issues as a test case to fix.

Reply:
I added an issue for this now that it is submitted.